modules/nodes-cluster-project-overcommit.adoc

```diff
@@ -6,10 +6,19 @@
 [id="nodes-cluster-project-overcommit_{context}"]
 = Project-level limits
 
+ifndef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 To help control overcommit, you can set per-project resource limit ranges,
 specifying memory and CPU limits and defaults for a project that overcommit
 cannot exceed.
 
 For information on project-level resource limits, see Additional resources.
 
 Alternatively, you can disable overcommitment for specific projects.
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+
+ifdef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+In {product-title}, overcommitment of project-level resources is enabled by default.
+If required by your use case, you can disable overcommitment on projects that are not managed by Red Hat.
+
+For the list of projects that are managed by Red Hat and cannot be modified, see _Red Hat Managed resources_ in Additional resources.
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations 


> Do not state "in the Additional resources section" or other variants